### ISSUE TITLE
Examples: update syntax and readme's

### ISSUE THREAD

**Issue by K33TY** [apl-cornell/fabric#8]
All the examples do not have readme files, and most are using the deprecated syntax. Some of them do not run correctly with persistence if nodes are shutdown and then restarted. Should fix all of this if planning on releasing next 'official' version of Fabric.

**Comment by liujed** [comment]
I think the real issue is that `examples/` has become a dumping ground for all sorts of random programs people have written for various purposes. Not everything in `examples/` is intended to be part of a public release. The `tests/playground` directory was intended to be the dumping ground for random programs, but that has been overlooked because it's lower in the directory hierarchy.

Modernizing the syntax and adding READMEs would be great, but I think someone should also go through and decide what should be a featured example, and what shouldn't. The ones in the latter category should either moved out of `examples` and into `tests/playground` or into another repo altogether.

**Comment by tmagrino** [comment]
Agreed with @liujed on needing to be a bit more discerning about what lives in examples.  I think one of the reasons many of these aren't being placed in tests/playground is because authors of some of these programs (I'm one of them ;) ) don't typically think of these programs as tests.  I'm guessing getting a proper runtime testing framework might change where a few of these programs should live, but I like the idea of moving some of the cruft over there for now.
